src/State/Hydration/reducers.ts

```diff
@@ -3,7 +3,10 @@ import * as types from './types';
 export const initialState = {
   isDestinationModalOpen: false,
   isEditorOpen: false,
-  selectedNode: null
+  selectedNode: null,
+  metaData: {
```

Comment:
metaData should be metadata, no need for camelcase. Not a big deal thought

Reply:
Updated on my other branch [https://github.com/pnmac-edma/kirby-react/pull/118]